configs/try-win64.cmd.user_data

```diff
@@ -0,0 +1,8 @@
+<script>
+sc delete puppet
+cscript.exe "C:\ProgramData\Puppetlabs\puppet\var\puppettize_TEMP.vbs"
+cd "C:\Program Files (x86)\Puppet Labs\Puppet\bin"
+cmd /c puppet agent --test --environment=mcornmesser --server=releng-puppet2.srv.releng.scl3.mozilla.com
+schtasks /delete /TN RUNPUPPET /F
+shutdown /s /t 0 /c "userdata run complete" /f /d p:4:1
+</script>
```

Comment:
I'm not an expert to review these files, but they look sane to me. Do you still need to use --environment here?